localization/english/dvg_names_l_english.yml

```diff
@@ -634,6 +634,10 @@ Lysandre: "Lysandre"
 Lavisse: "Lavisse"
 Gaius: "Gaius"
 Odysseus: "Odysseus"
+Falcone: "Falcone"
```

Comment:
Duplicate localization key. Key 'Falcone' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/dvg_names_l_english.yml'.